src/app/containers/MediaPlayer/index.jsx

```diff
@@ -173,35 +179,41 @@ const MediaPlayerContainer = ({
       <Caption block={captionBlock} type={mediaInfo.type} service={service} />
     ) : null;
 
+  const mediaPlayerType = isAmp ? (
```

Comment:
I think the var name `mediaPlayer` would be more appropriate. I'd expect `mediaPlayerType` to be a string value `amp` or `canonical`

Reply:
makes sense, I changed this too.
Thank you 👍